source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -104,6 +129,232 @@ TEST_F(NiRFmxWLANDriverApiTests, InitializeFromNIRFSA_Close_Succeeds)
   ni::tests::system::EXPECT_SUCCESS(close_response);
 }
 
+TEST_F(NiRFmxWLANDriverApiTests, OFDMModAccTXPCompositeFromExample_FetchData_DataLooksReasonable)
+{
+  auto session = init_session(stub(), PXI_5663E);
+  EXPECT_SUCCESS(session, client::cfg_frequency_reference(stub(), session, "", FREQUENCY_REFERENCE_SOURCE_ONBOARD_CLOCK, 10e6));
+  EXPECT_SUCCESS(session, client::cfg_frequency(stub(), session, "", 2.412e9));
+  EXPECT_SUCCESS(session, client::cfg_reference_level(stub(), session, "", 0.0));
+  EXPECT_SUCCESS(session, client::cfg_external_attenuation(stub(), session, "", 0.0));
+  EXPECT_SUCCESS(session, client::cfg_iq_power_edge_trigger(stub(), session, "", "0", IQ_POWER_EDGE_TRIGGER_SLOPE_RISING_SLOPE, -20.0, 0.0, TRIGGER_MINIMUM_QUIET_TIME_MODE_AUTO, 5.0e-6, IQ_POWER_EDGE_TRIGGER_LEVEL_TYPE_RELATIVE, BOOLEAN_TRUE));
+  EXPECT_SUCCESS(session, client::cfg_standard(stub(), session, "", STANDARD_802_11_AG));
+  EXPECT_SUCCESS(session, client::cfg_channel_bandwidth(stub(), session, "", 20e6));
+  EXPECT_SUCCESS(session, client::select_measurements(stub(), session, "", MEASUREMENT_TYPES_TXP|MEASUREMENT_TYPES_OFDMMODACC, true));
+  EXPECT_SUCCESS(session, client::ofdm_mod_acc_cfg_measurement_length(stub(), session, "", 0, 16));
+  EXPECT_SUCCESS(session, client::ofdm_mod_acc_cfg_averaging(stub(), session, "", OFDM_MOD_ACC_AVERAGING_ENABLED_FALSE, 10));
+  EXPECT_SUCCESS(session, client::txp_cfg_averaging(stub(), session, "", TXP_AVERAGING_ENABLED_FALSE, 10));
+  EXPECT_SUCCESS(session, client::txp_cfg_maximum_measurement_interval(stub(), session, "", 1e-3));
+  EXPECT_SUCCESS(session, client::initiate(stub(), session, "", ""));
+
+  OFDMModAccFetchCompositeRMSEVMResponse ofdm_mod_acc_fetch_composite_rmsevm_response;
```

Comment:
These are declared separately because their codegenerated? I'll allow it. That's fine.

Reply:
I intentionally generated variable declarations at the top of the block in which they're first used. If you'd rather they all be at the top of the function, I can do that.
(The examples define all the variables at the top of the function because they're in C [vs. C++], which requires that.)

Reply:
Oh no, this is way better than the C examples! That style is a nightmare for transcription for one thing.

My preference is:

```c++
const auto ofdm_mod_acc_fetch_composite_rmsevm_response = client::ofdm_mod_acc_fetch_composite_rmsevm(stub(), session, "", 10.0);
```

That allows it to be const and all-on-one-line. I think of it as avoiding the extra default initialization, but that might be optimizer-dependent anyway (though some of that is more official in c++17).

Reply:
Ah, right. I agree. I had another look at the generator, and it was easy enough to change. @reckenro I pushed another change that can be merged in, to inline the declarations when possible. You (or I) can probably just include it in a future update.